odl/tomo/backends/astra_cuda.py

```diff
@@ -39,12 +39,7 @@ class AstraCudaProjectorImpl(object):
 
     """Thin wrapper around ASTRA."""
 
-    algo_id = None
-    vol_id = None
-    sino_id = None
-    proj_id = None
-
-    def __init__(self, geometry, reco_space, proj_space, use_cache):
+    def __init__(self, geometry, reco_space, proj_space):
```

Comment:
Overall I have no idea why we did not do this before. Isn't this a strict improvement in readability and maintainability? For example I noticed a "small" memory "leak" (we forgot to remove the in and out arrays in the backprojection) which lost me 1gb of memory and this was trivially fixed by changing this.